docs/admin/addons.rst

```diff
@@ -207,10 +203,10 @@ Statistics generator
 
 Generates a file containing detailed info about the translation status.
 
-You can use Django template in both filename and content, see :ref:`markup`
+You can use a Django template in both filename and content, see :ref:`markup`
```

Comment:
one/the/a?

Reply:
a is right

Reply:
```suggestion
You can use a Django template for both filename and content, see :ref:`markup`
```
? (assumption is that the Django template isn't in there)